docs-src/concepts/what-is-a-workflow-execution.md

```diff
@@ -145,15 +145,16 @@ For example, it may be reasonable to use Continue-As-New once per day for a long
 
 ### Limits
 
-Each pending Activity generates a metadata entry in the Workflow's mutable state.
+There is no limit to the number of concurrent Workflow Executions.
+
+Each in-progress Activity generates a metadata entry in the Workflow's mutable state.
 Too many entries create a large mutable state, which causes unstable persistence.
+To protect the system, Temporal enforces a maximum number (2,000 by default) of incomplete Activities, Child Workflows, Signals, or Cancellation requests per Workflow Execution.
```

Comment:
It is unclear if this limit is aggregate across all these or per type. Is it OK to have 2k activities and 2k child workflows at the same time.

Reply:
Clarified by @lorensr to "2000 for each type of operation"